src/resolvers/projectResolver.ts

```diff
@@ -1799,17 +1799,12 @@ export class ProjectResolver {
   ): Promise<ProjectUpdatesResponse> {
     const latestProjectUpdates = await ProjectUpdate.query(`
       SELECT pu.id, pu."projectId"
-      FROM project_update as pu
-      WHERE pu.id = (
-        SELECT puu.id
-        FROM project_update as puu
-        WHERE puu."isMain" = false AND pu."projectId" = puu."projectId"
-        ORDER BY puu."createdAt" DESC
-        LIMIT 1
-      )
-      ORDER BY pu."createdAt" DESC
+      FROM public.project_update AS pu
+      WHERE pu."isMain" = false
+      GROUP BY pu.id, pu."projectId"
```

Comment:
Are you sure it must be grouped by both `id` and `projectId`?! maybe just `projectId`

Reply:
You are right Amin I removed the double grouping. I only need the projectUpdate.Id